Podfile

```diff
@@ -2,10 +2,10 @@ platform :osx, '10.9'
 use_frameworks!
 
 def common_pods
-  pod 'SwiftGenKit', '~> 1.1.0'
+  pod 'SwiftGenKit', :git => 'git@github.com:SwiftGen/SwiftGenKit.git', :branch => 'feature/parser-protocol'
```

Comment:
⚠️  Reminder not to forget to change that back once we have merged https://github.com/SwiftGen/SwiftGenKit/pull/46